### PR TITLE
Create Add-a-Business-Bank-Account-(AUD).md

### PR DESCRIPTION
**Pull request by bfitzexpensify**
### Details
Adding a new article to ExpensifyHelp via the instructions in https://stackoverflowteams.com/c/expensify/questions/15135

### Fixed Issues

$https://github.com/Expensify/Expensify/issues/309817

### CODE REVIEW

**Comment by bfitzexpensify** [comment]
OK, comments removed @rushatgabhane

File: docs/articles/expensify-classic/bank-accounts-and-credit-cards/business-bank-accounts/Add-a-Business-Bank-Account-(AUD).md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Deep Dive
+## Bank-specific batch payment support
+
+If you are new to using Batch Payments in Australia, to reimburse your staff or process payroll, you may want to check out these bank-specific instructions for how to upload your .aba file:
+
+ANZ Bank - [Import a file for payroll payments](https://www.anz.com.au/support/internet-banking/pay-transfer-business/payroll/import-file/)
+CommBank - [Importing and using
 Direct Entry (EFT) files](https://www.commbank.com.au/business/pds/003-279-importing-a-de-file.pdf)
+Westpac - Importing Payment Files[https://www.westpac.com.au/docs/pdf/bb/Importing-Payment-Files-Guide-Online.pdf)
```
Comment (youssef-lr) [inline docs/articles/expensify-classic/bank-accounts-and-credit-cards/business-bank-accounts/Add-a-Business-Bank-Account-(AUD).md:39]: > https://www.westpac.com.au/docs/pdf/bb/Importing-Payment-Files-Guide-Online.pdf)!

This link seems dead?

**Comment by bfitzexpensify** [comment]
Good catch @youssef-lr - I've updated that link now

**Review by rushatgabhane — APPROVED** [review]
@youssef-lr LGTM

**Review by youssef-lr — APPROVED** [review]

**Comment by youssef-lr** [comment]
Not a deploy blocker!

**Pull request merged**